pyproject.toml

```diff
@@ -31,7 +30,8 @@ urls.changelog = "https://build.pypa.io/en/stable/changelog.html"
 urls.homepage = "https://build.pypa.io"
 urls.issues = "https://github.com/pypa/build/issues"
 urls.source = "https://github.com/pypa/build"
-
+# Version read from __version__ field in __init__.py by Flit
+dynamic = ["version"]
```

Comment:
Is flit clever about this or will it require installing all of our runtime deps?

Reply:
I'm pretty sure it's clever about it. But good point, let me check.

Reply:
https://github.com/pypa/flit/blob/8ead22d4d3028f2dd2c952c85e5bf1535cca3b40/flit_core/flit_core/common.py#L140